### 4.8 单行对象括号之间保留空格

- BAD

```javascript
const handlesomer = { name: 'ranwawa', age: 18 };
```

- GOOD

```javascript
const handlesomer = { name: 'ranwawa', age: 18 };
```

- WHY
  - 操作符两边以及逗号之后会有一个空格
  - 保持括号之间有空格更加统一
