### 4.7 多行对象结尾加逗号

- BAD

```javascript
const handsomer = {
  name: 'ranwawa',
  age: 18
};
```

- GOOD

```javascript
const handsomer = {
  name: 'ranwawa',
  age: 18
};
```

- WHY
  - 保持统一,避免 git 冲突
